### DISCUSSION TITLE
Waydroid

### DISCUSSION THREAD

**Discussion by compata** [appium/appium#19220] [Q&A]
Appium setup instructions (I'm a beginner with appium) seem to imply that one of the first steps is to create a new Android emulator. But I already have one, Waydroid. Is it possible to use appium with Waydroid? If so, are there any online instructions?

**Comment by KazuCocoa** [comment]
Asking such questions at https://discuss.appium.io/ might help. I haven't used that

**Reply by compata** [reply]
Thanks. I've tried to do that but that site doesn't seem to be working correctly. It requires signup, which I've tried to do, but the confirmation email never arrives. I do know how to check my mail server logs and no message from them has ever even been attempted.

**Reply by jlipps** [reply]
I think the only signup method that works might be github sso?
